aries/aries_vcx_core/src/errors/mapping_anoncreds.rs

```diff
@@ -0,0 +1,41 @@
+use anoncreds::{Error as AnoncredsError, ErrorKind};
```

Comment:
Shall we submit rename the error type in `anoncreds-rs`? 

Would be easier to work with - the alias helps, but it's actually confusing for reader still. For example intellisense shows me `cause` variable on line `7` to be of type `Error` (defined in `anoncreds-rs`), but the downcasting refers to type `AnoncredsError`. Altough obivously it's the same thing.

Reply:
It is not uncommon to name error types plainly `Error`, so I don't see submitting an error rename (a breaking change) as justified. (Unlike intellisense) I don't see type aliasing as inherently problematic, although in these cases full qualification (`anoncreds::Error`) is clearly better choice.

Reply:
You are right, looking at bunch of high profile crates that seems to be the practice.